lms/djangoapps/courseware/access.py

```diff
@@ -550,14 +554,18 @@ def can_load():
         students to see modules.  If not, views should check the course, so we
         don't have to hit the enrollments table on every module load.
         """
+        # Masquerading as a learner in a group needs to work in preview mode, and
```

Comment:
This is a little confusing to read as it is assuming that the reader knows about this bug. I'd frame it more like this:

    If the user does not have access to this content then deny access. Note that this check obeys
    masquerading so that staff can preview content as if they are a learner.